qa/rolling-upgrade/src/test/java/org/opensearch/neuralsearch/bwc/NeuralQueryEnricherProcessorIT.java

```diff
@@ -117,11 +126,6 @@ public void testNeuralQueryEnricherProcessor_NeuralSearch_E2EFlow() throws Excep
                 loadModel(denseModelId);
                 neuralQueryBuilderWithModelId.modelId(denseModelId);
 
-                createSearchRequestProcessor(denseModelId, DENSE_SEARCH_PIPELINE_NAME);
-                updateIndexSettings(
```

Comment:
can you please add more context on why exactly these two lines are not needed anymore?

Reply:
Because that was repetitive code. This 2 lines were called in testAgainstOldCluster and again used in mixed cluster.

Reply:
got it, it's not directly related to the main issue, but more like a side fix